src/components/attribute-input/AttributeInput.tsx

```diff
@@ -58,63 +58,95 @@ export const AttributeInput = ({
     }
   }, []);
 
-  const [isOpenArray, setIsOpenArray] = useState<boolean[]>([false]);
+  const [isKeyOpenArray, setIsKeyOpenArray] = useState<boolean[]>([false]);
   const watchLastKey = watch(`${name}[${fields.length - 1}].key`, "");
   const watchLastValue = watch(`${name}[${fields.length - 1}].value`, "");
 
-  const [valueOpen, setValueOpen] = useState(false);
-  const toggleSelect = (rowIndex: number, open: boolean) => {
-    const arr = [...isOpenArray];
+  const [isValueOpenArray, setIsValueOpenArray] = useState<boolean[]>([false]);
```

Comment:
I think Typescript can infer the type here from the assigned value.
```suggestion
  const [isValueOpenArray, setIsValueOpenArray] = useState([false]);
```